lib/refresh.js

```diff
@@ -99,7 +97,7 @@ const handleDrop = function(evt, ui, originalIndex, originalOrder) {
 
   if (moveWithinPage) {
     const order = getStoryItemOrder($item.parents('.story:first'));
-    if (!_.isEqual(order, originalOrder)) {
+    if (arrayShallowEqual(order, originalOrder)) {
```

Comment:
Missing `!`?

Reply:
Paul and I added some console.log to confirm that the lists are lists of strings. This from decaffeinate branch without your change.

<img width="289" alt="image" src="https://github.com/user-attachments/assets/89f515dc-727d-4202-8cb4-1d197838fb8a">

Reply:
That is a very good question Paul, I don't know why it seemed to work like this :sweat_smile:

Reply:
Oh it definitely does not work, I expected to see a problem while dropping, not having to refresh.

Reply:
I recommend we apply toString() to these arrays and then test for string inequality.
<img width="404" alt="image" src="https://github.com/user-attachments/assets/0671b46d-1560-453d-86e6-9bd76898dd69">
With Nick's permission I will push this change to his branch.

Reply:
Whoops, I just pushed a fix using `JSON.stringify` which seemed to be working well

Reply:
It's pretty niche, but `.toString()` could get tripped up if someone generated pages with a `,` in the item ids, but `JSON.stringify()` shouldn't
```
> JSON.stringify([ 1, 2, 3, undefined, null, "4,5"])
'[1,2,3,null,null,"4,5"]'
> [ 1, 2, 3, undefined, null, "4,5"].toString()
'1,2,3,,,4,5'
> 
```